projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -333,6 +338,14 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         return this._dropDownOverlaySettings || this._defaultDropDownOverlaySettings;
     }
 
+    public get required(): boolean {
```

Comment:
Not sure why this is still a public getter when it's not used by a template ( should've be hidden before). Make it private?

Reply:
actually it will be @hidden @internal
Otherwise I should remove all `Control value accessor unit tests` as this is what they are testing :) (or cast `datePicker` to `any` all the time)

Reply:
Those can just as easily test the inputGroup `isRequired` is assigned properly, no?